### DISCUSSION TITLE
How to move wallpaper directory

### DISCUSSION THREAD

**Discussion by KevTheGamer412** [rocksdanister/lively#519] [General]
When i moved lively to my d drive (because of lack of storage on c drive) The wallpapers that came with lively are gone, and when i deleted lively and redownloaded it it still wasnt there. Please fix this bug

**Comment by rocksdanister** [comment]
I don't fully understand what you mean by move to d (whether you installed the program on D:\\ or moved the wallpaper folder from lively settings to D:\\ or both..)

If you changed the wallpaper directory then just change it back to the old location `C:\Users\<UserName>\AppData\Local\Lively Wallpaper\Library` and this time check "Move existing wallpaper.." and then move to D:\\ again.
![Screenshot 2021-05-25 223411](https://user-images.githubusercontent.com/17554161/119539097-71205880-bda9-11eb-9c63-e960401eac23.png)

**Comment by erwinwallpaper** [comment]
Thanks.
